ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -81,6 +81,7 @@ conformance_test(
         ",ParticipantPruningIT" +  # see "conformance-test-participant-pruning" below
         ",ClosedWorldIT" +  # Canton currently fails this test with a different error (missing namespace in "unallocated" party id)
         ",CommandServiceIT:CSsubmitAndWaitCompletionOffset" +  # Canton does not fill the offsets
+        ",CommandServiceIT:CSsubmitAndWaitForTransactionBasicEmptyLedgerId" +  # Canton requires non-empty ledger id
```

Comment:
This only holds until Canton is upgraded to the new release.

A way to solve this without undue configuration overhead and juggling is to introduce an experimental feature descriptor and guard the test case using that one. This way you are done after this PR while keeping backwards compatibility of the test tool.

Reply:
Ok, trying the feature descriptor